eng/vcpkg-commit.txt

```diff
@@ -1 +1 @@
-be5c4ef68b51142ba705f0678b45d284977de677
+c556911a79f8ab23b12648ddfe8a5444a6c05080
```

Comment:
Unintended change?

Reply:
no, we have this amazing thing in the cmakegenerationbuild steps  that checks out vcpkg to a specific commit , that is the SHA of the commit to which vcpkg will sync. the previous version was before the release of core 1.5, the second is after 1.5 was released.

Reply:
FWIW, I'm hoping to get rid of this entirely, with @danieljurek 's help.

Reply:
In my PR, this entire thing is redundant, instead we use the baseline-commit feature of the vcpkg.json file to handle this.